keps/sig-cli/0015-kubectl-plugins.md

```diff
@@ -0,0 +1,152 @@
+---
+kep-number: 15
+title: Kubectl Plugins 
+authors:
+  - "@juanvallejo"
+  - "@soltysh"
```

Comment:
@soltysh you sure you want credit here instead of being an approver?

Reply:
agree, author+approver is weird

Reply:
approver, I'll be hidden author ;)